src/qcodes/dataset/descriptions/dependencies.py

```diff
@@ -290,6 +290,11 @@ def _empty_data_dict(self) -> dict[str, dict[str, npt.NDArray]]:
         for standalone in (ps.name for ps in self.standalones):
             output[standalone] = {}
             output[standalone][standalone] = np.array([])
+        # Add inferred parameters as their own trees
+        for inferred in self.inferences.keys():
+            inferred_name = inferred.name
+            if inferred_name not in output:
+                output[inferred_name] = {inferred_name: np.array([])}
         return output
```

Comment:
Currently things fail because this does not produce the same data structure as get_parameter_data